deploy/docker/start_backend.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Wait until RTS started and listens on port 8091
+while [ -z "`curl localhost:8091/rts-api/v1/health-check`" ]; do
```

Comment:
Instead of checking the output, let's see if the request is successful, that is, responds with a 200 status code. We can use the `--fail` argument for this.

```suggestion
while ! curl --fail --silent localhost/rts-api/v1/health-check; do
```